.travis.yml

```diff
@@ -3,16 +3,20 @@ node_js: 10.19.0
 dist: trusty
 services:
   - docker
-before_install:
-- docker pull backstopjs/backstopjs:3.9.2
```

Comment:
Moving this into just the testing job saves us ~40 seconds for our other jobs that don't need it